deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -85,6 +85,9 @@ spec:
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
+{{- range $key, $value := .Values.cniCustomPodAnnotations }}
```

Comment:
Follow the same conventions as Istio's other charts, e.g. [Pilot's chart](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml#L38-L40):
```
         {{- if .Values.podAnnotations }}
{{ toYaml .Values.podAnnotations | indent 8 }}
        {{- end }}
```

Also use the same name `podAnnotations` for consistency.

Reply:
Also, all Helm values must be documented with their default values in the `values*.yaml` files, preferably with a comment explaining the semantics of the parameter. E.g. for [Pilot's chart](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/pilot/values.yaml#L30):
```
podAnnotations: {}
```